GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -688,11 +687,15 @@ void ExporterHelper::AddLibsInclude(bool pixiRenderers,
   }
   if (pixiRenderers) {
     InsertUnique(includesFiles, "pixi-renderers/pixi.js");
+    InsertUnique(includesFiles, "pixi-renderers/pixi-spine.js");
```

Comment:
I think there is no need to include this for all games. Instead, you can use `.setIncludeFile(`/`.addIncludeFile(` on the object (`addObject`) of the extension :)

Reply:
hm. Then I need to do it somehow for `pixi-atlas-manager` and `pixi-json-manager` (probably I can refactor it as I mention in comment for `spine` resource)

Reply:
Yes I think ideally we can also do it for resource managers (it's a bit less important: I'm fine having extra, useless resource managers in all games, even if it's a bit suboptimal) (but I want to for sure avoid having pixi-spine entirely if it's not needed, which is what the extensions system is designed for)